src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRISE_PER_DAY = 40;
```

Comment:
There is a typo in the constant name `PRISE_PER_DAY`. It should be `PRICE_PER_DAY` to correctly reflect its purpose.